hack/conformance.sh

```diff
@@ -12,11 +12,25 @@ fi
 export KUBECONFIG=$(cluster-up/kubeconfig.sh)
 
 teardown() {
+    rv=$?
```

Comment:
What is rv? could you give this variable more informative name?

Reply:
return value, its pretty common in bash

Reply:
Please consider renaming thin value to `returnValue` or  `errorCode`,
even though it is common in bash it is not trivial for new comers.

Reply:
Please consider renaming thin value to returnValue or errorCode,
even though it is common in bash it is not trivial for new comers.